libnavui-speedlimit/src/main/res/values-ru/strings.xml

```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources xmlns:tools="http://schemas.android.com/tools">
+    <string name="max_speed">Скорость\n%1$d</string>
+    <string name="max_speed_no_value" tools:ignore="TypographyDashes">Скорость\n--</string>
```

Comment:
Capturing from https://github.com/mapbox/mapbox-navigation-android/pull/3838#discussion_r540544634

> I think it should be localizable on Android with a note to make the translation as short as possible, so it fits in the design.

@RingerJK @korshaknn @alexander-kulikovskii @truburt could you confirm this is the shortest option in Russian? 🙏

Reply:
`Скорость` translates as `Speed`. If we want to say `Max Speed` it would be `Максимальная Cкорость`. I think the presented options(`Скорость\n--` and `Скорость\n%1$d`) are the shortest. We should keep them

Reply:
btw in Russian(and CIS countries I think) no sign `Max speed` with words, it's always a circle sign, red borders, and speed limit

![image](https://user-images.githubusercontent.com/8738058/126501093-ea956658-556d-41ca-a5e6-543b032f1d25.png)

Reply:
@RingerJK 
> `Скорость` translates as `Speed`. If we want to say `Max Speed` it would be `Максимальная Cкорость`. I think the presented options(`Скорость\n--` and `Скорость\n%1$d`) are the shortest. We should keep them

Should it be translated as `MAX` instead of speed? How would that be? That's ultimately what we're translating https://github.com/mapbox/mapbox-navigation-android/blob/65a66fd79fc8a96cbb21ae4cdff5d58b95b28dc6/libnavui-speedlimit/src/main/res/values/strings.xml#L3-L4

Reply:
> btw in Russian(and CIS countries I think) no sign `Max speed` with words, it's always a circle sign, red borders, and speed limit

Yeah, logic is handled internally, so no `MAX` will be shown 👍

Reply:
I haven't seen `MAX` in local navigators, so I think it would be wired to write it. We can actually replace it with `Лимит`(en `Limit`). It's very close to the meaning

Reply:
Have you seen `Speed` (`Скорость`) in local navigators? If so, we can leave it as is. Was only flagging 👍

Reply:
>Have you seen Speed (Скорость) in local navigators?

No, I haven't, it's always like https://github.com/mapbox/mapbox-navigation-android/pull/4649#discussion_r674000716. I'd set red borders for SpeedLimitView without words at all for the Russian location

Reply:
> I'd set red borders for SpeedLimitView without words at all for the Russian location

Yeah, as I mentioned that's already handled. We should be good.